riotfile.py

```diff
@@ -1609,7 +1609,7 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
             },
             env={
                 "DD_AGENT_PORT": "9126",
-                "DD_PYTEST_USE_NEW_PLUGIN_BETA": "0",
+                "DD_PYTEST_USE_NEW_PLUGIN_BETA": "1",
```

Comment:
Should this be configured by customers? If so, can we add it to `configuration.rst`?

Reply:
Yes, this variable was made public in 2.18.0 (https://github.com/DataDog/dd-trace-py/pull/11431/files#diff-0991608ac3b38804a08a91fd4ace9098bb705da0dc5133c912b75ce98ed2a0f7R1). It will become deprecated in 3.x, but I can add a note for it in `configuration.rst`.

Reply:
@erikayasuda Updated documentation in https://github.com/DataDog/dd-trace-py/pull/11828/commits/c7c3319b94a411265128ff0be518cc3d281c6301, please have a look!